completion_aggregator/management/commands/run_aggregator_service.py

```diff
@@ -24,12 +24,43 @@ class Command(BaseCommand):
     run_aggregator_service management command.
     """
 
+    def add_arguments(self, parser):
+        """
+        Add command-line arguments
+        """
+        parser.add_argument(
+            '--routing-key',
+            dest='routing_key',
+            help='Celery routing key to use.',
+        )
+        parser.add_argument(
+            '--batch-size',
+            help='Maximum number of CourseModuleCompletions to migrate, per celery task. (default: 10000)',
+            default=100000,
+            type=int,
+        )
+        parser.add_argument(
+            '--delay-between-batches',
+            help='Amount of time to wait between processing batches in seconds.  (default: 0.0)',
+            default=0.0,
+            type=float,
+        )
+        parser.add_argument(
+            '--limit',
+            help='Amount of time to wait after submitting each 1000 tasks.  (default: 0.0)',
```

Comment:
This help text is wrong and the `limit` parameter doesn't seem to be used - it doesn't get passed to `perform_aggregation`